src/routes/events/+page.svelte

```diff
@@ -4,10 +4,10 @@
 	export let data;
 </script>
 
-<h1>Events</h1>
+<h1>The Latest and Greatest CodeSeoul Events!</h1>
 
 <div>
-	{#if data.events.length === 0}
+	{#if data.events && data.events.length === 0}
```

Comment:
I think it'd be better to set the condition as `{#if !data.events || data.events.length === 0}` so we can also control the case when events doesn't exist inside data (I'm not sure if this is a possibility)

Reply:
Ah! Good catch